dart/lib/src/sentry_tracer.dart

```diff
@@ -0,0 +1,20 @@
+import '../sentry.dart';
+import 'tracing.dart';
+
+class SentryTracer extends SentrySpan {
+  Hub _hub;
+  SentryTransactionContext _transactionContext;
+
+  // missing waitForChildren
+
+  String? name;
+  @override
+  late SentrySpanContext context;
+  DateTime? timestamp;
+  DateTime? startTimestamp;
+  late bool isFinished;
+
+  // find out how to pass its own instance to super
+  SentryTracer(this._hub, this._transactionContext)
+      : super(_transactionContext);
```

Comment:
@ueman do you know if this is even possible?

Reply:
I don't see why it shouldn't be possible.

Though maybe it has to look like this:
```dart
SentryTracer(this._hub, SentryTransactionContext transactionContext)
      : _transactionContext = transactionContext, super(transactionContext);
```